tests/e2e_logging.py

```diff
@@ -264,6 +265,53 @@ def test_update_lua(network, args):
     return network
 
 
+@reqs.description("Check for commit of every prior transaction")
+@reqs.supports_methods("getCommit", "tx")
+def test_view_history(network, args):
```

Comment:
Should this be run against all nodes of the network?

Reply:
Good idea, it is now.